pinot-core/src/main/java/org/apache/pinot/core/operator/filter/BitmapBasedFilterOperator.java

```diff
@@ -97,6 +100,11 @@ protected FilterBlock getNextBlock() {
       if (_exclusive) {
         docIds.flip(0L, _numDocs);
       }
+      InvocationRecording recording = Tracing.activeRecording();
+      if (recording.isEnabled()) {
+        recording.setNumDocsMatchingAfterFilter(docIds.getCardinality());
```

Comment:
Should we also record the column name? We can get the `Predicate` from the `PredicateEvaluator`

Reply:
done